components/Document/pages/DocumentPageLayout.tsx

```diff
@@ -55,18 +60,24 @@ const DocumentPageLayout = ({
     console.log("Error building open graph data", e);
   }
 
-  const pdfUrl = document.formats.find((f) => f.type === "pdf")?.url;
   return (
-    <div className={css(styles.pageWrapper)}>
+    <div className={css(styles.pageWrapper) + " " + documentPageClass}>
       <HeadComponent {...openGraphData.meta} graph={openGraphData.graph}>
         <meta
           name="viewport"
           content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no"
         />
       </HeadComponent>
 
-      <div className={css(styles.topArea)}>
-        <DocumentHeader document={document} metadata={metadata} />
+      <div className={css(styles.topArea) + " " + topAreaClass}>
```

Comment:
I mentioned in slack, I think that the post should have a minimal header component that doesn't have voting, tip, and all the bells and whistles. That way, we can keep the integrity of the `<DocumentHeader>` component

Reply:
I don’t think we should build another component here. The post is a researchhub specific post so it should render very similar to how the post renders on our main site